src/domain/position/index.js

```diff
@@ -36,7 +36,8 @@ const changeParticipantPosition = (participantCurrencyId, isReversal, amount, tr
 }
 
 const calculatePreparePositionsBatch = async (transferList) => {
-  return await PositionFacade.prepareChangeParticipantPositionTransaction(transferList)
+  let result = await PositionFacade.prepareChangeParticipantPositionTransaction(transferList)
```

Comment:
Why is the result assigned to a variable and then returned? Could it not be done in one line with return await?